### PR TITLE
1.11.4 Release Notes

### PR DESCRIPTION
**Pull request by anithamuthu79**
https://jira.mesosphere.com/browse/DCOS_OSS-3854

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by anithamuthu79** [comment]
@catsouthard @pavisandhu @yishan-lin 
1.11.4 RN PR is ready for review. During your review, please check and comment on the lines for missing ENT tags. Also, check for internal changes that should be removed from the release notes. 

@vinodkone @kensipe @artemharutyunyan @meichstedt @alenkacz @ichernetsky please review and provide the correct change log links for Mesos, Marathon and Metronome.

**Comment by anithamuthu79** [comment]
Reminder - @vinodkone @kensipe @artemharutyunyan @meichstedt @alenkacz @ichernetsky please review and provide the correct change log links for Mesos, Marathon and Metronome.

File: pages/1.11/release-notes/1.11.4/index.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+DC/OS 1.11.4 was released on July 26, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.4/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.4 includes the following:
+
+- Apache Mesos 1.5.2-dev [change log](https://github.com/mesosphere/mesos/blob/197c9105c2ed08f1529602f1ff72de846cde98c6/CHANGELOG).
+- Marathon 1.6.496 [change log](https://github.com/dcos/dcos/blob/1.11.4/packages/marathon/buildinfo.json).
```
Comment (orsenthil) [inline pages/1.11/release-notes/1.11.4/index.md]: `marathon-1.6.535`

Comment (orsenthil) [reply pages/1.11/release-notes/1.11.4/index.md]: Please change the URL to https://github.com/mesosphere/marathon/releases/tag/v1.6.535

File: pages/1.11/release-notes/1.11.4/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+DC/OS 1.11.4 was released on July 26, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.4/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.4 includes the following:
+
+- Apache Mesos 1.5.2-dev [change log](https://github.com/mesosphere/mesos/blob/197c9105c2ed08f1529602f1ff72de846cde98c6/CHANGELOG).
```
Comment (vinodkone) [inline pages/1.11/release-notes/1.11.4/index.md]: Let's point to https://github.com/apache/mesos/blob/2440c7310627c835fe828aef6b28b8d072e7371c/CHANGELOG

this sha matches what's in the `buildinfo.json` here https://github.com/dcos/dcos/blob/1.11.4/packages/mesos/buildinfo.json

Comment (anithamuthu79) [reply pages/1.11/release-notes/1.11.4/index.md]: @vinodkone Done.

**Pull request merged**